apps/web/src/styles/toast.css

```
@@ -105,7 +220,163 @@
+  padding: 0;
+  font-family: var(--font-display), sans-serif;
+  font-size: clamp(1rem, 2.8vw, 1.125rem);
+  font-weight: 700;
+  line-height: 1.25;
+  letter-spacing: -0.03em;
+  color: var(--toast-update-headline);
+}
+
+.toast-update-support {
+  margin: 0 0 14px;
+  font-family: "AirbnbCereal", sans-serif;
```

Stylelint (17.7.0)

[error] Expected no quotes around "AirbnbCereal" (font-family-name-quotes)

(font-family-name-quotes)